pcsx2/GS/GSUtil.cpp

```diff
@@ -190,14 +190,30 @@ CRCHackLevel GSUtil::GetRecommendedCRCHackLevel(GSRendererType type)
 
 GSRendererType GSUtil::GetPreferredRenderer()
 {
-#ifdef __APPLE__
+#if defined(__APPLE__)
+	// Mac: Prefer Metal hardware.
 	return GSRendererType::Metal;
-#endif
-#ifdef _WIN32
+#elif defined(_WIN32)
+#if defined(ENABLE_OPENGL)
+	// Windows: Prefer GL if available.
```

Comment:
```suggestion
	// Windows: Prefer DX11 if available.
```

Reply:
Why? GL is better..

Reply:
No. DX11 has the worst accuracy on windows. If it wasn't for old GPUs, I'd make Vulkan the default. But regardless, the goal of this PR isn't to change the defaults.